test_unstructured/partition/test_api.py

```diff
@@ -210,6 +211,52 @@ def test_partition_via_api_valid_request_data_kwargs():
     assert isinstance(elements, list)
 
 
+def test_partition_via_api_image_block_extraction():
+    filename = os.path.join(DIRECTORY, "..", "..", "example-docs", "embedded-images-tables.pdf")
+    elements = partition_via_api(
+        filename=filename,
+        strategy="hi_res",
+        extract_image_block_types=["image", "table"],
+        api_key=get_api_key(),
+    )
+    image_elements = [el for el in elements if el.category == ElementType.IMAGE]
+    for el in image_elements:
+        assert el.metadata.image_base64 is not None
+        assert el.metadata.image_mime_type is not None
+        image_data = base64.b64decode(el.metadata.image_base64)
+        assert isinstance(image_data, bytes)
```

Comment:
additionally, we could try to load the image with PIL and check some basic metadata to confirm it's a correct image file. But as this is a high-priority bug - we could do that in a separate PR.

Reply:
Sounds reasonable! I'll plan to do that in a separate PR.